### PR TITLE
fix: correctly normalize intentionally blank pages

### PR DESCRIPTION
**Pull request by eventualbuddha**

### CODE REVIEW

File: src/Interpreter.test.ts [hunk 1]
```diff
@@ -3468,3 +3469,26 @@ test('choctaw 2020 general', async () => {
     ]
   `)
 })
+
+test('normalizes intentionally empty pages correctly', async () => {
+  const fixtures = choctaw2020Special
+  const { election } = fixtures
+  const interpreter = new Interpreter(election)
+
+  await interpreter.addTemplate(await fixtures.blankPage1.imageData())
```
Comment (benadida) [inline src/Interpreter.test.ts:3478]: is this a blank first page? Should it be a normal first page to be a true test?

**Review by benadida — APPROVED** [review]
one question regarding the test.

File: src/Interpreter.test.ts (hunk 1 above)
Comment (eventualbuddha) [reply src/Interpreter.test.ts:3478]: The terminology is a little inconsistent. This "blank" means "unmarked", as in a template page.

Comment (benadida) [reply src/Interpreter.test.ts:3478]: ah got it, so the other page is actually empty / no-contests? cool.

**Pull request merged**